DFU: implemented LED signal for DFU state (#332)
nrfconnect/sdk-sidewalk: doc/testing_samples/sidewalk_testing.rst

************************************

#. To enter the DFU mode, long press **Button 4** on your development kit.

>    This action restarts the device in DFU mode where Sidewalk is not operational, and only `Zephyr SMP Server`_ is running.

Reviewer: @RobertGalatNordic Don't think any of the suggested changes was applied

>    This action restarts the device in DFU mode where Sidewalk is not operational, and only `Zephyr SMP Server`_ is running.

Author: oh yes, I made another change locally, and pushed it to PR  efectivelly overwriting what has been added :( my mistake

   When application is in DFU mode, all LED's will flash every 500 ms to signal that it is waiting for new image.

#. To perform a firmware update, follow the Bluetooth testing steps from the `DevZone DFU guide`_.
   When the update starts, the LED's will change the pattern to indicate upload in progress.

#. To exit the DFU mode, reset your device.
   The device will restart in the Sidewalk mode.
   If the update completes successfully, the device will start a new image.

>    First boot up after DFU may take more time than usual (usualy about 60 seconds). During the image swap the application is silent, so be paitient and do not reset the device.

Reviewer: ```suggestion
   After using the DFU mode, the first bootup might take up to 60 seconds. 
   During the image swap the application is silent, meaning that LEDs are not blinking and the console does not output any logs. 
   The application, however, is running, so wait and do not reset the device.
```

>    However, if the update fails, the old image will be started.

Reviewer: ```suggestion
   In case the update fails, the old image is started instead.
```
